style-guide/how-to-article-guidelines.rst

```diff
@@ -0,0 +1,500 @@
+  one word for a particular object. Technical writing is not creative writing,
+  and you should not be concerned that you will bore customers with colorless
+  prose. Clarity is the goal, so using a precise set of terms consistently is
+  required. Following is a common example of multiple terms that refer to the
+  same thing:
+
+  - menu command *(the preferred term)*
+  - menu item
+  - menu option
+
+- Use a word as only one part of speech. Many words can be correctly used as a
+  verb and as a noun or an adjective, such as *display*. However, using the
+  same word as more than one part of speech in the same document can be
+  confusing to customers and translators, so avoid it when possible.
+
+- Avoid fabricated words. Examples of fabricated words are *marketecture* or
+  *edutainment*. Most such words are specific to a single business culture and
+  are not understood in other cultures.
+
+- Standardize words and spelling across a documentation set.
+
+- Don't use terms with different meanings interchangeably. Some terms have
+  similar but distinct meanings and should not be used interchangeably. For
+  example:
+
+  - environment, platform
+  - version, release
+  - panel, screen
+  - window, dialog box
+
+Additionally, replace the following nonpreferred terms with the preferred
+terms.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Avoid
+     - Preferred
+   * - allow
+     - enable or you can (unless referring to permissions)
+   * - as
+     - because (describing a reason for)
+   * - below
+     - following (when referring to the placement of information "on the page")
+   * - box
+     - computer (or an more specific term, such as server)
+   * - checkbox
+     - check box
+   * - click on
+     - click
+   * - config
+     - configuration
+   * - deselect
+     - clear
+   * - desire
+     - want
+   * - desired
+     - wanted, correct, appropriate
+   * - double click
+     - double-click
+   * - e.g.
+     - for example
+   * - enter in
+     - enter
+   * - etc.
+     - and so on (or revise to omit)
+   * - i.e.
+     - that is
+   * - info
+     - information
+   * - information on
+     - information about
+   * - internet
+     - internet
+   * - log into
+     - log in to
+   * - may
+     - might (could happen) or can (is able to)
+   * - machine
+     - computer (except when discussing a virtual machine, or VM)
+   * - once
+     - after (to mean occurring subsequently in time or order)
+   * - make
+     - sure	ensure
+   * - please
+     - Omit. It is not necessary.
+   * - refer to
+     - see
+   * - right click
+     - right-click
+   * - since
+     - because (describing a reason for)
+   * - using
+     - by using
+
+For more guidelines about terminology, see the following sections in the
+style guide:
+
+- `Terminology for a global audience <https://developer.rackspace.com/docs/style-guide/terminology/terms-for-global-audience/#terms-for-global-audiencee>`__
```

Comment:
It looks like there's an extra "e" right before the closing bracket.

Reply:
Fixed